easybuild/scripts/bootstrap_eb.py

```diff
@@ -448,7 +459,9 @@ def main():
     for path in orig_sys_path:
         include_path = True
         # exclude path if it's potentially an EasyBuild package
-        if 'easybuild' in path:
+        if 'easybuild' in path or os.path.exists(os.path.join(path, 'easybuild')):
+            include_path = False
+        if '/vsc_' in path or os.path.exists(os.path.join(path, 'vsc')):
```

Comment:
excluding any entries in sys.path that look like they have something to do with VSC tools (vsc-base, or something else)

the `/` and `_` are there specifically to recognize `.egg` dirs, mostly

`vsc` by itself seems a bit too generic to exclude paths, it *may* be a partial match for a username, for example